src/Types.ts

```diff
@@ -54,19 +54,29 @@ type RegionInfo = {
 }
 
 // Available color sets
-enum ColorPallete{
+enum ColorPalette{
   greys,
   ygreys,
   blues,
   reds,
   plainColors,
-  lightColors
+  lightColors,
+}
+
+// Hex describing a color
+type colorHEX = `#${string}`;
```

Comment:
Maybe this should be called something like `ColorHexCode` or `ColorHex` to match the formatting of the other types?